common/client-core/src/client/packet_statistics_control.rs

```diff
@@ -486,7 +486,7 @@ impl PacketStatisticsControl {
             // Check what the number of retransmissions was during the recording window
             if let Some((_, start_stats)) = self.history.front() {
                 let delta = self.stats.clone() - start_stats.clone();
-                log::info!(
+                log::debug!(
```

Comment:
Nice, I've been meaning to bump these down to debug, and instead emit events that the vpn client can listen for so we don't pollute the log output